recycler/src/main/java/com/elpassion/android/commons/recycler/strategies/items/impl/MapItemsStrategy.kt

```diff
@@ -0,0 +1,72 @@
+package com.elpassion.android.commons.recycler.strategies.items.impl
+
+import com.elpassion.android.commons.recycler.strategies.items.MutableSectionedItemsStrategy
+import com.elpassion.android.commons.recycler.strategies.items.SectionedItemsStrategy
+import java.util.*
+
+class MapItemsStrategy<SectionType, ItemType>(list: Map<SectionType, List<ItemType>>) : SectionedItemsStrategy<SectionType, ItemType> {
+
+    private val items: Map<SectionType, List<ItemType>> = LinkedHashMap<SectionType, List<ItemType>>(list.size).apply {
+        putAll(list.mapValues { ArrayList(it.value) })
+    }
+
+    override fun allItems() = items.values.flatten()
+
+    override fun getSection(section: SectionType) = items[section]!!.toList()
```

Comment:
@OrdonTeam I wouldn't, @langara is right, in my opinion we should return just null in such case so the user knows that there is no such section